.github/workflows/post-announce.yml

```diff
@@ -0,0 +1,60 @@
+name: Bump action runner version
+
+on:
+  workflow_call:
+    inputs:
+      plan:
+        required: true
+        type: string
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    env:
+      PLAN: ${{ inputs.plan }}
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Plan details
+        run: |
+          echo "Plan details: $PLAN"
+
+      - name: Check if runner was released
+        id: check_runner
+        run: |
+          IS_PRE_RELEASE=$(echo ${PLAN} | jq '.announcement_is_prerelease')
+          NEW_VERSION=$(echo ${PLAN} | jq -r '.releases[] | select(.app_name == "codspeed-runner") | .app_version')
+          RELEASE_TAG=$(echo ${PLAN} | jq -r '.announcement_tag')
+
+          echo "is_pre_release=${IS_PRE_RELEASE}" >> "$GITHUB_OUTPUT"
+          echo "new_version=${NEW_VERSION}" >> "$GITHUB_OUTPUT"
+          echo "release_tag=${RELEASE_TAG}" >> "$GITHUB_OUTPUT"
+
+          if [ "${IS_PRE_RELEASE}" == "true" ]; then
+            echo "runner_released=false" >> "$GITHUB_OUTPUT"
+            echo "Pre-release detected, will not mark as latest or bump action"
+          elif [ -z "${NEW_VERSION}" ] || [ "${NEW_VERSION}" == "null" ]; then
+            echo "runner_released=false" >> "$GITHUB_OUTPUT"
+            echo "No codspeed-runner release found in plan"
+          else
```

Comment:
I think in this case, we still need to publish the release but without putting it as latest. WDYT?